contract-ui/tabs/overview/components/BuildYourApp.tsx

```diff
@@ -0,0 +1,47 @@
+import { ChakraNextImage as Image } from "../../../../components/Image";
+import { Flex, GridItem, SimpleGrid } from "@chakra-ui/react";
+import { PRODUCTS } from "components/product-pages/common/nav/DesktopMenu";
+import { Card, Heading, Link, Text } from "tw-components";
+
+const RENDERED_PRODUCTS = [0, 7, 8, 1];
```

Comment:
this looks a bit weird to me, can we define them by name or something and then find them in the array rather than hardcoding the index number that can potentially change in the future?

Reply:
we can use the label